b3_3@sirius-NUC13ANHi5:~/tanaka$ python3 0228_cb.py 
List Trainer: 0it [00:00, ?it/s]
Traceback (most recent call last):
  File "/home/b3_3/tanaka/0228_cb.py", line 11, in <module>
    trainer.train(training_)
  File "/home/b3_3/.local/lib/python3.10/site-packages/chatterbot/trainers.py", line 117, in train
    self.chatbot.storage.create_many(statements_to_create)
  File "/home/b3_3/.local/lib/python3.10/site-packages/chatterbot/storage/sql_storage.py", line 283, in create_many
    raise Exception('generate bulk_search_text values')
Exception: generate bulk_search_text values
